Source/CarthageKit/Project.swift

```diff
@@ -348,8 +348,8 @@ public final class Project {
 	///
 	/// Sends the URL to each downloaded zip, after it has been moved to a
 	/// less temporary location.
-	private func downloadMatchingBinariesForProject(project: ProjectIdentifier, atRevision revision: String, fromRepository repository: GitHubRepository, withCredentials credentials: GitHubCredentials?) -> SignalProducer<NSURL, CarthageError> {
-		return releaseForTag(revision, repository, credentials)
+	private func downloadMatchingBinariesForProject(project: ProjectIdentifier, atRevision revision: String, fromRepository repository: GitHubRepository, withAuthorizationHeaderValue authorizationHeaderValue: String? = .None) -> SignalProducer<NSURL, CarthageError> {
```

Comment:
Is there any benefit of the default argument for `withAuthorizationHeaderValue`? This method is called only in two places above, so explicitly passing nil is better I think.